### PR TITLE
update places docs

### PR DESCRIPTION
**Pull request by sbenedicadb**

### CODE REVIEW

File: foundation-extensions/places/README.md [hunk 1]
```diff
@@ -1,13 +1,179 @@
...
+### Java
+
+```java
+public class MobileApp extends Application {
+
+    @Override
+    public void onCreate() {
+      super.onCreate();
+      MobileCore.setApplication(this);
+      MobileCore.configureWithAppID("yourLaunchEnvironmentID");
```
Comment (PravinPK) [inline foundation-extensions/places/README.md:111]: can we remove the launch reference here. and do something like
MobileCore.configureWithAppID("yourAppID");

File: foundation-extensions/places/README.md [hunk 2]
```diff
@@ -1,13 +1,179 @@
...
+Adobe Experience Platform Location Service provides an SDK extension which allows you to act based on the location of your users. This extension is the interface to the [Location Service Web Services APIs](https://experienceleague.adobe.com/docs/places/using/web-service-api/places-web-services.html?lang=en).
 
 The SDK extension listens for events that contain GPS coordinates and geofence region events, and dispatches new events that are processed by the Rules Engine. The SDK extension also retrieves and delivers a list of the nearest POI for the app data that retrieves from the APIs. The regions returned by the APIs are stored in cache and persistence, which allows limited offline processing.
 
-To get started with implementing Adobe Experience Platform Places Service on your mobile app, review documentation on [Places extension](https://experienceleague.adobe.com/docs/places/using/places-ext-aep-sdks/places-extension/places-extension.html).
+`Places` is the mobile SDK supporting the Location Service.
 
-## Reference
+## Add the AEP Places extension to your app
```
Comment (PravinPK) [inline foundation-extensions/places/README.md:9]: Before getting to configuring Places SDK in app
Should we write about 
`Configuring Places extension in Data Collection UI` ?

File: foundation-extensions/places/places-usage-reference.md [hunk 3]
```diff
@@ -2,6 +2,12 @@
 
 This document contains usage information for the public functions, classes, and enums in AEPPlaces.
 
+{% hint style="info" %}
+This page only contains information about the 3.x `AEPPlaces` extension for iOS.
+
+A full API reference for the Android `Places` extension and 2.x `ACPPlaces` extension for iOS can be found [here](https://experienceleague.adobe.com/docs/places/using/places-ext-aep-sdks/places-extension/places-api-reference.html?lang=en).
+{% endhint %}
+
 ## Static functions
```
Comment (PravinPK) [inline foundation-extensions/places/places-usage-reference.md:12]: Nit:  It will be a good idea to port release notes as well

**Review by calebk1 — COMMENTED** [review]
Looks good!

Not really any net-new comments here.

File: SUMMARY.md [hunk 4]
```diff
@@ -76,10 +76,10 @@
   * [Release Notes](foundation-extensions/adobe-experience-platform-assurance/adobe-experience-platform-assurance-release-notes.md)
   * [Assurance API reference](foundation-extensions/adobe-experience-platform-assurance/assurance-api-reference.md)
   * [Resolving Common Issues](foundation-extensions/adobe-experience-platform-assurance/assurance-sdk-error-logs.md)
-* [Adobe Experience Platform Places Service](foundation-extensions/places/README.md)
+* [Adobe Experience Platform Location Service](foundation-extensions/places/README.md)
   * [Migrating to AEPPlaces reference](foundation-extensions/places/migration.md)
   * [Places API reference](foundation-extensions/places/places-usage-reference.md)
-  * [ACPPlaces \(iOS\) reference](foundation-extensions/places/adobe-places.md)
+  * [Additional Location Service resources](foundation-extensions/places/adobe-places.md)
```
[the remark this answers is not in the document]
Comment (sbenedicadb) [reply SUMMARY.md:82]: not that i'm aware of

[the remark this answers is not in the document]
Comment (calebk1) [reply SUMMARY.md:82]: Got it! Will keep that in mind.

File: foundation-extensions/places/README.md (hunk 2 above)
Comment (sbenedicadb) [reply foundation-extensions/places/README.md:19]: added

**Review by calebk1 — APPROVED** [review]
LGTM!

**Review by PravinPK — APPROVED** [review]

**Pull request merged**